### PR TITLE
Malaysia building

### PR DESCRIPTION
**Pull request by conroy-chee**
Initialize Malaysia dbo

*Changes on HVAC ABSTRACT.yaml, AHU.yaml, FCU.yaml, GENERALTYPES.yaml, SENSOR.yaml and VAV.yaml
*Changes on SAFETY ABSTRACT.yaml and LDS.yaml
*Added HVAC VRV.yaml and METERS FM.yaml

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml [hunk 1]
```diff
@@ -263,6 +263,16 @@ VLV:
   implements:
   - EQUIPMENT
 
+VRV:
+  id: "15255671887819388960"
+  description: "A single split unit."
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml:268]: If this is a variable refrigerant flow system (VRF) please describe it as such.

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 2]
```diff
@@ -745,6 +775,72 @@ FCU_DFMSS_DSP_ZHM_RTM_CHWZC_CHWDT:
...
+    - supply_fan_speed_percentage_sensor
+  implements:
+    - FCU
+    - CHWDC
+    - DFVSC
+
+FCU_ETIQA_CRAC:
+  id: "10065218199902748673"
+  description: "Malaysia ETIQA Crac device"
+  is_canonical: true
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:828]: If this is a strange one-off device, mark it as non-canonical.

**Review by tasodorff — CHANGES_REQUESTED** [review]

**Comment by charbull** [comment]
@conroy-chee please fix the following errors: 

Findings in SAFETY/entity_types/LDS.yaml
ERROR : Duplicate type IDs are not allowed. Entity type name "LDS_GASDET" within namespace "SAFETY" with ID "17602426483400245248" was already defined in 
	<SAFETY/entity_types/LDS.yaml> (Line 0).

ERROR : Duplicate type IDs are not allowed. Entity type name "LDS_BC2X_WLA_LPA" within namespace "SAFETY" with ID "17602426483400245248" was already defined in 
	<SAFETY/entity_types/LDS.yaml> (Line 0).

Findings in HVAC/entity_types/VRV.yaml
ERROR : Duplicate type IDs are not allowed. Entity type name "VRV_ZTC_VSC" within namespace "HVAC" with ID "13434279712669433856" was already defined in 
	<HVAC/entity_types/VRV.yaml> (Line 0).

ERROR : Duplicate type IDs are not allowed. Entity type name "VRV_NO_CONTROL" within namespace "HVAC" with ID "13434279712669433856" was already defined in 
	<HVAC/entity_types/VRV.yaml> (Line 0).

Link errors: https://github.com/google/digitalbuildings/pull/262/checks?check_run_id=2936487827

Thank you

**Comment by tasodorff** [comment]
@conroychee is this still in progress.

**Pull request closed without merging**